Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -460,7 +461,10 @@ def on_metadata_received_alert(self, alert):
                 self.metadata["announce"] = trackers[0]
 
         self.tdef = TorrentDef.load_from_dict(self.metadata)
+
+        # files() is iterable
```

Comment:
Please, don't add white-space modifications or irrelevant commits to the PR, it will cause unnecessary merge conflicts to others.